packages/AutoTDD-Core.package/ATDDGitHubActionWorklowJob.class/class/newForId.andName..st

```diff
@@ -0,0 +1,7 @@
+as yet unclassified
+newForId: aNumber andName: aString
```

Comment:
I think it is uncommon in Squeak to use "and" for additional parameters. Please check other methods to verify. Otherwise refactor to:

```suggestion
newForId: aNumber name: aString
```